app/scripts/util/focus-detector.js

```diff
@@ -0,0 +1,20 @@
+const FeatureDetector = require('./feature-detector');
+const Launcher = require('../comp/launcher');
+
+const FocusDetector = function () {
+    this.isFocused = false;
```

Comment:
I don't think this default is the proper value, changed it to `true` because it broke initial focus when you have recent files. Please check if the fix is still working.